modules/ROOT/pages/custom-user-information-in-bonita-bpm-applications.adoc

```diff
@@ -1,4 +1,5 @@
 = Custom User Information Management
+:page-aliases: custom-user-information-in-bonita-bpm-portal.adoc
```

Comment:
This page can even be renamed `custom-user-information-in-bonita-applications.adoc`
Now that the product name does not include BPM anymore

Reply:
Renamed it